client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -4,15 +4,14 @@
  */
 import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
-import { get, map, partial, pickBy, startsWith, isArray, flowRight } from 'lodash';
```

Comment:
Not related to the PR, but I just wanted to say I'm happy to see lodash usage being removed 😉